autogpt_platform/backend/backend/copilot/briefing/outcome_test.py

```
@@ -0,0 +1,141 @@
+    ) == ("Sorted 12 emails.", "Nothing needed a reply.")
+
+
+def test_run_link_needs_a_library_agent() -> None:
+    assert run_link(None, "execution") is None
+    assert run_link("library agent", "exec/1") == (
+        "/library/agents/library%20agent?activeTab=runs&activeItem=exec/1"
+    )
+
+
+def test_as_utc_pins_naive_timestamps() -> None:
+    assert as_utc(datetime(2026, 8, 10, 9, 0)) == NOW
```

Ruff (0.16.1)

[warning] `datetime.datetime()` called without a `tzinfo` argument

(DTZ001)